test/resources/serial_console.resource

```diff
@@ -0,0 +1,43 @@
+*** Settings ***
+Library     PexpectLibrary
+Library    serial_console.py
+
+Resource    serial_console.resource
+
+
+*** Variables ***
+${PROMPT_PATTERN}   \\[["\\w"@\\-~\\ ]+\\]\\$\\ *
+
+*** Keywords ***
+Send Command
+    [Arguments]     ${command}
+    Send Line    sudo ${command}
+    ${index}=   Expect    ${{ [$PROMPT_PATTERN, pexpect.TIMEOUT, pexpect.EOF] }}   timeout=60s
+    Should Be Equal As Integers     ${index}    0
+
+Login To Guest
+    ${index}=   Expect   ${{ ['^.*login:', r'$PROMPT_PATTERN', pexpect.TIMEOUT, pexpect.EOF] }}
+    Should Not Be Equal As Integers     ${index}    2
+    Should Not Be Equal As Integers     ${index}    3
+    IF    ${index} == 0
```

Comment:
The case for index = 1 isn't handled.

What do all of these index values mean? What's special about non-zero values?